packages/components/src/Components/FormElements/InputCheckbox/InputCheckbox.js

```diff
@@ -142,25 +142,33 @@ const InputCheckbox = props => {
 };
 
 InputCheckbox.propTypes = {
+  checked: PropTypes.bool,
```

Comment:
In general, try to make these changes in a separate PR, they aren't really related with updating create-react-app :)